x/farming/spec/04_messages.md

```diff
@@ -9,7 +9,7 @@ Messages (Msg) are objects that trigger state transitions. Msgs are wrapped in t
 Anyone can create this private plan type message. 
 
 - A fixed amount plan distributes the amount of coins by a fixed amount that is defined in `EpochAmount`. 
-- Internally, `PrivatePlanFarmingPoolAddress` is generated and assigned to the plan. 
+- Internally, The Private Plan Farming Pool Address is derived and assigned to the plan. 
```

Comment:
```suggestion
- Internally, the private plan's farming pool address is derived from the following derivation rule of `address.Module(ModuleName, []byte("PrivatePlan|{planId}|{planName}"))` and it is assigned to the plan. 
```

Reply:
`address.Module(ModuleName, []byte("PrivatePlan|{planId}|{planName}"))` is already written on concepts.md, and state_transitions.md, so let me leave this line as `- Internally, the private plan's farming pool address is derived and assigned to the plan. ` for simple